### ISSUE TITLE
::Rake.Application.last_comment needs to be removed

### ISSUE THREAD

**Issue by cdenneen** [rspec/rspec-core#2205]
```
.bundle/ruby/2.1.0/gems/rspec-core-2.99.2/lib/rspec/core/rake_task.rb:143:        desc "Run RSpec code examples" unless ::Rake.application.last_comment
```

```
 bundle exec rake lint
rake aborted!
NoMethodError: undefined method `last_comment' for #<Rake::Application:0x00000000dbffb0>
/var/lib/jenkins/jobs/test/workspace/profile_base/.bundle/ruby/2.1.0/gems/rspec-core-2.99.2/lib/rspec/core/rake_task.rb:143:in `initialize'
/var/lib/jenkins/jobs/test/workspace/profile_base/.bundle/ruby/2.1.0/gems/puppetlabs_spec_helper-1.1.1/lib/puppetlabs_spec_helper/rake_tasks.rb:15:in `new'
/var/lib/jenkins/jobs/test/workspace/profile_base/.bundle/ruby/2.1.0/gems/puppetlabs_spec_helper-1.1.1/lib/puppetlabs_spec_helper/rake_tasks.rb:15:in `<top (required)>'
/var/lib/jenkins/jobs/test/workspace/profile_base/Rakefile:2:in `require'
/var/lib/jenkins/jobs/test/workspace/profile_base/Rakefile:2:in `<top (required)>'
(See full trace by running task with --trace)
```

rake 10.5.0 has alias for last_comment to last_description but this was removed in 11.0

```
.bundle/ruby/2.1.0/gems/rake-10.5.0/lib/rake/task_manager.rb:10:    alias :last_comment :last_description # :nodoc: Backwards compatibility
```

**Comment by mrageh** [comment]
@cdenneen RSpec 2.99 was only released to help users upgrade to RSpec 3, please refer to this PR https://github.com/rspec/rspec-core/pull/2202 for more information.

**Comment by darkhelmet** [comment]
I'm seeing this on rspec-core-3.5.0.beta1

**Comment by darkhelmet** [comment]
Looks like this is fixed in 8e723fc805e901ac4fa5483837138b175d411d6e

**Comment by mjankowski** [comment]
Any chance of getting a `3.5.0.beta2` released for people running the rspec beta alongside rails betas?

**Comment by JonRowe** [comment]
`master` will always be the most recent version but I've just released a `3.5.0.beta2`

**Comment by c-beltran** [comment]
what I did was added the gem 'rake', '11.1.0', and then updated rake "bundle update rake"
